h5p-bildetema/src/hooks/useSiteLanguage.ts

```diff
@@ -1,5 +1,31 @@
+import { Language } from "common/types/types";
+import { useNewDBContext } from "common/hooks/useNewDBContext";
+import { attributeLanguages } from "common/constants/languages";
 import { useL10n } from "./useL10n";
 
-export const useSiteLanguage = (): string => {
+export const useSiteLanguageString = (): string => {
   return useL10n("htmlLanguageCode");
 };
+
+export const useSiteLanguage = (): Language => {
+  const { languages } = useNewDBContext();
+  const siteLanguageString = useSiteLanguageString();
+
+  const siteLanguage = languages.find(
+    lang =>
+      siteLanguageString ===
+      (attributeLanguages as Record<string, string>)[lang.code],
+  );
+
+  if (!siteLanguage) {
+    return { code: "nob", label: "Bokmål", rtl: false } as Language;
```

Comment:
The "Bokmål" might be problematic with UTF-8 changing between the VMS. Is Bokmål in the database?

Reply:
Never had a problem with it